docs/DeveloperGuide.md

```diff
@@ -278,43 +278,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ---
 
-**Use Case: UC1 - Listing all Persons**
-
-**MSS**
-
-1. User requests to view all contacts in the contact list.
-2. Spend n Split shows a list of contacts.
-
-Use case ends.
-
-Extensions:
-* 1a. The user can include a flag to sort the list by name.
-  * 1a1. Spend n Split shows the list of contacts now sorted by name.
-  * 1a2. Use case resumes at step 4.
-
----
-
-**Use Case: UC2 - Adding a New Person**
+**Use Case: UC1 - Adding a New Person**
 
 **MSS**
 
 1. User requests to add a new person.
 2. Spend n Split informs the user that the new person has been added.
-3. Spend n Split shows the updated contact list.
+3. Spend n Split shows the updated contacts list.
```

Comment:
I think "contact list" is more appropriate here, but I'm okay with both, just gotta standardise throughout the use cases and DG

Reply:
I think person list is used by the app

Reply:
Updated